com.md

```diff
@@ -0,0 +1,33 @@
+# Subject: Proposal for new website
+
+**To: server-user@james.apache.org, server-dev@james.apache.org**
+
+Hello,
+
+We would like to propose some website improvements. They include:
+
+ - A new responsive, modern and appealing homepage for the James project
+ - We keep the main website the same but:
+   - We improve side menu navigation with a single side menu, all across the website
+   - Make top menu consistent
+   - We revisited some critical pages, and tried to make it more consistent
+   - Remove most stable
+   - We re-wrote pages about performance, migration support, etc...
+   - Landing page of old website have been made more user friendly.
+ - News have been moved to homepage
+
+The improvement of the website is an important milestone towards James 3.0 release.
+
+You can view the result online: http://62.210.100.33
+
+Now we will:
+
+ - Welcome feed-backs for one week (until 21/04/2017 10 am UTC+1)
```

Comment:
![screenshot from 2017-04-14 07-48-49](https://cloud.githubusercontent.com/assets/6928740/25029373/db94d394-20e6-11e7-81d5-7ee4ea0ad9e6.png)

France. It happens being the afternoon in Vietnam.